core/node/events/miniblock_info.go

```diff
@@ -249,16 +253,19 @@ func (b *MiniblockInfo) lastEvent() *ParsedEvent {
 	}
 }
 
-func (b *MiniblockInfo) IsSnapshot() bool {
-	return b.Header().GetSnapshot() != nil || len(b.Header().GetSnapshotHash()) > 0
+// IsSnapshot returns non-nil but empty byte array if the miniblock is a snapshot.
+// TODO: Will be replaced with the real snapshot after DB migration. WIP.
+func (b *MiniblockInfo) IsSnapshot() []byte {
+	if b.Header().GetSnapshot() != nil || len(b.Header().GetSnapshotHash()) > 0 {
+		return make([]byte, 0)
+	}
+
+	return nil
 }
```

Comment:
weird that isSnapshot nolonger returns bool - should we change this to GetSnapshot()?